core/src/main/resources/hudson/PluginManager/installed.jelly

```diff
@@ -68,9 +68,9 @@ THE SOFTWARE.
       <table id="plugins" class="pane bigtable sortable stripped-odd">
         <j:choose>
           <j:when test="${empty(app.pluginManager.plugins) and empty(app.pluginManager.failedPlugins)}">
-            <tr><td>
+            <tr><th>
```

Comment:
shouldn't this just be a regular cell?

the dark background looks quite weird in the screenshot?

i.e. skip the header and just make it a td?

Reply:
I had the same thought, this should probably look similar to the placeholder text on the revised "Available" tab.

Reply:
The problem is that the first table row will look like a header. So then either we remove that message from the table or we do something like 

<img width="689" alt="Captura de pantalla 2020-08-03 a las 10 03 58" src="https://user-images.githubusercontent.com/5738588/89160060-c10b0600-d570-11ea-81c9-43087be893ce.png">

WDYT?

Reply:
👍

Reply:
Done

Reply:
@fqueiruga What's wrong with keeping the real table header and having a placeholder item with the message? Would that be confusing?

Reply:
Nothing, it's what I changed. I just didn't do it at the beginning because it didn't occur to me.

Reply:
Ah, with real table header you mean with the proper fields like name, version, etc?

Reply:
Yes, doesn't seem like it needs a special header unless that makes it clearer what's going on.

Very minor comment, the current state is already good 👍

Reply:
See my edit with what the code would look like